chia/wallet/wallet_nft_store.py

```diff
@@ -157,6 +157,19 @@ async def count(self, wallet_id: Optional[uint32] = None, did_id: Optional[bytes
                 return int(count_row[0])
         return -1
 
+    async def is_empty(self, wallet_id: Optional[uint32] = None) -> bool:
```

Comment:
if we have a unit test for `wallet_nft_store`, we should probably have a few more cases to cover this new function

Reply:
ah, yes